src/compute_horde_prompt_solver/run.py

```diff
@@ -91,8 +98,22 @@ def process_file(
         json.dump(responses, f, indent=2)
 
 
+def mock_run(input_file: pathlib.Path, output_dir: pathlib.Path):
+    num_prompts = 0
+    with open(input_file, "r") as f:
+        num_prompts = len([0 for line in f if line.strip()])
+
+    output_file = output_dir / f"{input_file.stem}.json"
+    with open(output_file, "w") as f:
+        json.dump(["mock mock mock" for _ in range(num_prompts)], f, indent=2)
```

Comment:
it should be a mapping from prompt to result - I think even if it is a mock it should keep the interface

Reply:
yeah